externs/browser/w3c_indexeddb.js

```diff
@@ -353,26 +418,20 @@ function IDBDatabase() {}
 IDBDatabase.prototype.name;
 
 /**
- * @type {string}
- * @const
- */
-IDBDatabase.prototype.description;
-
-/**
- * @type {string}
+ * @type {number}
  * @const
  */
 IDBDatabase.prototype.version;
 
 /**
- * @type {DOMStringList}
+ * @type {!DOMStringList}
  * @const
  */
 IDBDatabase.prototype.objectStoreNames;
 
 /**
  * @param {string} name The name of the object store.
- * @param {Object=} opt_parameters Parameters to be passed
+ * @param {!IDBObjectStoreParameters=} opt_parameters Parameters to be passed
```

Comment:
`?Object` was last specced in [Working Draft 2011](https://www.w3.org/TR/2011/WD-IndexedDB-20110419/) (7 versions before final recomendation). Again, I don't believe we can meet a browser with this.